lib/pages/chat_list/chat_list_view_style.dart

```diff
@@ -1,8 +1,17 @@
-import 'package:flutter/cupertino.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
+import 'package:flutter/material.dart';
 
 class ChatListViewStyle {
+  static final responsiveUtils = getIt.get<ResponsiveUtils>();
+
   static const editIconSize = 18.0;
 
   static Size preferredSizeAppBar(BuildContext context) =>
       const Size.fromHeight(120);
+
+  // Between 0 and 1, scale on actions length
+  static const double slidableExtentRatio = 0.25;
+  static const Color slidableForegroundColorDefault = Colors.white;
+  static const Color pinSlidableColorRaw = Color(0xFF00C853);
```

Comment:
why use this color?

Reply:
slidable background color is not nullable so I prefer to implement a safety check
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/a488afa3-386b-45ae-a89d-c11391fdd532)